src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathSingleFilterTest.java

```diff
@@ -492,6 +492,47 @@ public void testUpdateFilterChecksSettingInRunTime() throws Exception {
                 .isTrue();
     }
 
+    /**
+     * This test is required to cover pitest mutation
+     * for reset of 'message' field in SuppressionXpathSingleFilter.
+     * This not possible to reproduce by natural execution of Checkstyle
+     * as config is never changed in runtime.
+     * Projects that use us by api can reproduce this case.
+     * We need to allow users to reset module property to default state.
+     *
+     * @throws Exception when there is problem to load Input file
+     */
```

Comment:
Ok, such type of tests already exists in our codebase so it is easy to understand the PR. But it is always good to add some information about mutation in PR description.

Reply:
I have updated the PR description. Please review. Thank you.